qa/TL0_python_self_test_TU/test.sh

```diff
@@ -8,6 +8,8 @@ source qa/setup_dali_extra.sh
 cd dali/test/python
 
 test_body() {
+    # workaround for the CI
+    put_of_libs
```

Comment:
Sorry, but lowercase `of` is hardly descriptive. Make it either `OF` or `optflow`.

Reply:
Done